packages/core/src/components/EditableTypography/EditableTypography.module.scss

```diff
@@ -26,6 +26,12 @@
     }
   }
 
+  .textarea {
+    max-height: inherit;
```

Comment:
Where is it defined?

Reply:
it is to allow the user to define a max-height that we can respect
if there is none, it will let it grow as much as needed

Reply:
If it's a feature it should be via a prop, not by className

Reply:
you're right, or instead user can commandeer the container and limit its size